src/lib/constants/dev-environment.js

```diff
@@ -19,7 +19,7 @@ export const DEV_ENVIRONMENT_DEFAULTS = {
 	title: 'VIP Dev',
 	multisite: false,
 	phpVersion: '7.4',
-	elasticsearchVersion: '7.5.1',
+	elasticsearchVersion: '7.10.1',
```

Comment:
Is 7.10.1 the earliest available ARM build? I'm asking cause we decided to lock it closer to prod?

Reply:
The first ARM-compatible version is `7.8`. Given that we're not using `7.8` in prod, I thought `7.10` would be a better fit.